variables.tf

```diff
@@ -48,16 +49,15 @@ variable "existing_watsonx_data_instance_crn" {
 
 variable "plan" {
   type        = string
-  description = "The plan that is required to provision the watsonx.data instance. Possible values are: 'lite' , 'lakehouse-enterprise' and `lakehouse-enterprise-mcsp` (only for au-syd region). [Learn more](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-getting-started)"
+  description = "The plan that is required to provision the watsonx.data instance. Possible values are: 'lite', 'lakehouse-enterprise' and 'lakehouse-enterprise-mcsp'. 'lite' is available in eu-de, jp-tok and eu-gb. 'lakehouse-enterprise' is available only in eu-de, us-east, us-south, jp-tok, and eu-gb. 'lakehouse-enterprise-mcsp' is available only in au-syd and ca-tor. [Learn more](https://cloud.ibm.com/docs/watsonxdata?topic=watsonxdata-getting-started)"
```

Comment:
Please recheck, I have mentioned in previous comment also that there are only two plans shown to the user in the console. 
![Uploading image.png…]()